ag_data/migrations/0002_agactiveevent.py

```diff
@@ -1,21 +0,0 @@
-# Generated by Django 2.2.11 on 2020-04-10 23:06
```

Comment:
This PR is deleting some migration files. I guess the migration files are merged into a new single file. Will this work without problem during migration on other people's environment?

Reply:
Thanks for pointing this out. The reason to rebuild the migrations is that after the PK change, Django could not create new migrations, as certain fields require default values.

Reply:
Got it!